src/components/NFTPortfolio/MainView.vue

```diff
@@ -307,6 +309,7 @@ export default {
     };
   },
   computed: {
+    ...mapGetters(['getAddress']),
```

Comment:
What is this for?

Reply:
It's quick fix; this page hasn't imported the mixin, resulting in 'getAddress' being undefined